addons/web/static/tests/views/list_view_tests.js

```diff
@@ -5928,6 +5928,48 @@ QUnit.module("Views", (hooks) => {
         assert.verifySteps(["notify"]);
     });
 
+    QUnit.test("duplicate one record", async function (assert) {
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: `<tree editable="top" sample="1"><field name="foo"/></tree>`,
```

Comment:
could you remove sample="1" here and in the test below?

Reply:
Done